index.d.ts

```diff
@@ -519,13 +519,10 @@ declare class GlightboxInit {
     version(): string;
 }
 
-/**
- * TODO
- * How to type this default function?
- * When trying to click imported file it goes to this exported function not the Options
- */
-export default function (options = {}) {
-    const instance = new GlightboxInit(options);
+export default function (options: Glightbox.Options) {
+    let instance: InstanceType<typeof GlightboxInit>;

```

Comment:
Can this not be written as:
```ts
export default function(options: GLightbox.Options) {
    const instance:InstanceType<typeof GlightboxInit> = new GlightboxInit(otions);
    instance.init();

    return instance;
}
```
Or was does that not work in this case?

Reply:
@gingerchew the ts compiler returns an error if I write it as `const instance = new Glightbox({...})`

```
const' initializer in an ambient context must be a string or numeric literal" and "(TS) Initializers are not allowed in ambient contexts.
```

Reply:
I have moved away from initializing the class, and typing the instance instead

Reply:
@gingerchew I have improved the types, now the intellsense is much better, can you review last commit 👍 

![image](https://github.com/biati-digital/glightbox/assets/17225564/56704260-42ce-463a-be86-0244ff8cdaa4)